source/apiVolontaria/apiVolontaria/managers.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class ActionTokenManager(models.Manager):
+    def filter(self, expired=None, *args, **kwargs):
```

Comment:
Really good point! I took a moment after i read your review to have a thought about it!

I think it's maybe good to let it as i did because :
 - If you put an other type than a boolean, the filter work and return nothing.
 - If we refactor the system of expiration in the future, this code will be more generic and will not need a refactor too.

We just have (i think) no value to check if the type is a boolean or not in this part of the code.

Reply:
Agreed. Since this is a Model Manager, I think we should put the boolean restriction elsewhere (such as a serializer).